packages/plugin-ext/src/main/node/paths/plugin-paths-service.ts

```diff
@@ -142,7 +142,7 @@ export class PluginPathsServiceImpl implements PluginPathsService {
 
     private async cleanupOldLogs(parentLogsDir: string): Promise<void> {
         // @ts-ignore - fs-extra types (Even latest version) is not updated with the `withFileTypes` option.
-        const dirEntries = await readdir(parentLogsDir, { withFileTypes: true });
+        const dirEntries = await readdir(parentLogsDir, { withFileTypes: true }) as string[];
```

Comment:
I believe the `await` is unnecessary?

Reply:
`readdir` returns `Promise<string[]>` here, TypeScript is confused because we don't respect the typings (see comment above).

`as string[]` applies to the result of the `(await ...)` expression.